event/events.go

```diff
@@ -47,12 +54,21 @@ func Multiplex(events ...EventEmitter) *multiplexedEvents {
 
 // The events struct has methods for working with events.
 type events struct {
-	eventSwitch *evts.EventSwitch
+	eventSwitch *go_events.EventSwitch
 }
 
 // Subscribe to an event.
-func (this *events) Subscribe(subId, event string, callback func(evts.EventData)) error {
-	this.eventSwitch.AddListenerForEvent(subId, event, callback)
+func (this *events) Subscribe(subId, event string,
+	callback func(txs.EventData)) error {
+	cb := func(evt go_events.EventData) {
+		eventData, err := mapToOurEventData(evt)
+		if err != nil {
+			panic(fmt.Sprintf("Failed to map go-events EventData to our EventData %v",
+				err))
```

Comment:
OK. Mixing async paradigms (error channel where it's currently using callbacks) is not a net-gain on clarity, so let's not go there.  Im good with logging it and dropping it.